updater/lib/dependabot/opentelemetry.rb

```diff
@@ -15,19 +16,69 @@ def self.should_configure?
     sig { void }
     def self.configure
```

Comment:
We talked about this offline, but we might want to consider calling `start_root_span` in `configure` and then using `current_span` as a less invasive way to try this out.

Reply:
Is there any reason for me to refactor it from how I currently implemented it?

Reply:
I thought we could make the changes less invasive if we didn't wrap everything in blocks, but it's mostly a stylistic change.

Reply:
I think I'd rather have the name of the span be the method name. Vs one root one to use everywhere